src/api/index.ts

```diff
@@ -1,7 +1,9 @@
 import axios, { AxiosError } from 'axios'
 
 const API = axios.create({
-  baseURL: process.env.API_URL ?? 'https://admin.dev.climatepolicyradar.org/api/',
+  baseURL:
+    import.meta.env.VITE_API_URL ??
+    'https://admin.dev.climatepolicyradar.org/api/',
```

Comment:
I actually reckon leave as just the `.env` variable and force any devs to set it properly. I feel that's preferable to having the dev API in code.

Reply:
Good shout :+1: